### PR TITLE
Announces PRs in IA if they contain the [admin] tag

### PR DESCRIPTION
**Pull request by nichlas0010**
As per usual, I have no clue whether this works, I just threw it together from various JS functions I could find online

### CODE REVIEW

**Comment by ThatLing** [comment]
Nice branch name

File: models/Github/GithubManager.js [hunk 1]
```diff
@@ -136,6 +136,8 @@ class GithubManager {
     for (var channel of discordSubsystem.getPrimaryGuild().channels.array()) {
       if (channel.id == config.discord_coder_channel || channel.id == config.discord_coder_public_channel) {
         channel.sendEmbed(embed);
+			if(channel.id == config.discord_important_admin_channel && payload.pull_request.title.search("[admin]") != -1)
```
Comment (ThatLing) [inline models/Github/GithubManager.js]: nich what the fuck

Comment (nichlas0010) [reply models/Github/GithubManager.js]: waht

Comment (nichlas0010) [reply models/Github/GithubManager.js]: oh yeah, atom and github can't agree on which tab is good

**Comment by AshCorr** [comment]
I swear to god nich, I will revoke your merge access

**Comment by nichlas0010** [comment]
:omegalul:

**Pull request merged**